t/cmd/mod.t

```diff
@@ -1,6 +1,6 @@
 #!./perl
 
-print "1..13\n";
+print "1..15\n";
```

Comment:
Whenever I encounter one of these in a core `.t` file I replace it with `done_testing` at the end of the file.

Reply:
That would be tricky in that file. It's too early to use `test.pl`

Reply:
Oh hmm... OK that's fair ;)